parm/fv3lam_post_avblflds.xml

```diff
@@ -1067,7 +1061,7 @@
        <shortname>WEASD_ON_SURFACE</shortname>
        <pname>WEASD</pname>
        <fixed_sfc1_type>surface</fixed_sfc1_type>
-       <scale>4.0</scale>
+       <scale>3.0</scale>
```

Comment:
@EricJames-NOAA Please keep the default scale as 4.0, The user may set higher or lower scale from his application's control file, e.g. fv3lam_rrfs.xml.